rollup.config.js

```diff
@@ -0,0 +1,205 @@
+import { resolve, dirname } from 'path'
```

Comment:
another curiosity question on including sourcemaps in the build. I know they are hit or miss if popular packages include them, but I especially find the useful in the projects I work on for debugging purposes. Your thoughts on them?

Reply:
I'll take care of this next week! :)